app/views/contexts/question.py

```diff
@@ -18,7 +18,7 @@ def build_question_context(rendered_block, form):
         answer_ids.append(answer["id"])
 
         if answer["type"] in ("Checkbox", "Radio"):
```

Comment:
for my understanding why there is no `Dropdown` here ?

Reply:
This is for types that support the details answer options which is essentially a nested field within a radio/checkbox option, which dropdowns don't support.
![image](https://user-images.githubusercontent.com/35296336/151546132-960e8f19-61fc-4df6-9d0e-7d8737422f3e.png)